Fix to install into same location as designated by patch engines-install-in-libdir-ssl.patch

Signed-off-by: Jay Monkman <jay.monkman@freescale.com>

diff -Naupr openssl-1.0.1e-orig/Configure openssl-1.0.1e/Configure
--- openssl-1.0.1e-orig/Configure	2013-11-11 11:47:20.763349345 -0600
+++ openssl-1.0.1e/Configure	2013-11-12 15:24:21.299692886 -0600
@@ -1870,7 +1870,7 @@ while (<IN>)
 		}
 	elsif	(/^#define\s+ENGINESDIR/)
 		{
-		my $foo = "$prefix/$libdir/engines";
+		my $foo = "$prefix/$libdir/ssl/engines";
 		$foo =~ s/\\/\\\\/g;
 		print OUT "#define ENGINESDIR \"$foo\"\n";
 		}
